### PR TITLE
Update remarkable

### PR DESCRIPTION
**Pull request by nabeards**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by nabeards** [comment]
I'm not sure what the error is that happened...

**Comment by nabeards** [comment]
My first PR for brew. Feel free to squash / simplify commit messages.

File: Casks/remarkable.rb [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
   homepage "https://remarkable.com/"
 
   livecheck do
     url "https://get-updates.cloud.remarkable.engineering/sparkle/reMarkableMacOs/Prod/appcast.xml"
-    regex(%r{reMarkableMacOs/Prod/(\d+)/reMarkable-(\d+(?:\.\d+)*)\.dmg}i)
+    regex(%r{reMarkableMacOs/(Dev|Prod)/\d+/reMarkable-(\d+(?:\.\d+)*)\.dmg}i)
     strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| "#{match[1]},#{match[0]}" }
+      page.scan(regex).map { |match| match[1].to_s }
     end
```
Comment (bevanjkay) [inline Casks/remarkable.rb:16]:
```suggestion
```
You should be able to remove the entire strategy block now that the version and download URL has been simplified.

**Review by bevanjkay — DISMISSED** [review]

**Review by nabeards — COMMENTED** [review]
Looks good to me! 🙂 I was able to install without issue.

**Comment by miccal** [comment]
Thank you @nabeards.

**Pull request merged**